components/appHeader.vue

```diff
@@ -45,4 +45,7 @@ const closeMobileMenu = () => {
   isMobileMenuOpen.value = false
   document.body.style.overflow = ''
 }
+
+
+
```

Comment:
```suggestion
```
Whats the reason to add empty lines here?

Reply:
I'm guessing I added something in that file but ended up removing and forgot to remove those lines